opentelemetry-resource-detectors/src/process.rs

```diff
@@ -25,29 +25,74 @@ impl ResourceDetector for ProcessResourceDetector {
             .map(|arg| arg.to_string_lossy().into_owned().into())
             .collect::<Vec<StringValue>>();
         Resource::builder_empty()
-            .with_attributes(vec![
-                KeyValue::new(
-                    opentelemetry_semantic_conventions::attribute::PROCESS_COMMAND_ARGS,
-                    Value::Array(cmd_arg_val.into()),
-                ),
-                KeyValue::new(
-                    opentelemetry_semantic_conventions::attribute::PROCESS_PID,
-                    id() as i64,
-                ),
-            ])
+            .with_attributes(
+                vec![
+                    Some(KeyValue::new(
+                        opentelemetry_semantic_conventions::attribute::PROCESS_COMMAND_ARGS,
+                        Value::Array(cmd_arg_val.into()),
+                    )),
+                    Some(KeyValue::new(
+                        opentelemetry_semantic_conventions::attribute::PROCESS_PID,
+                        id() as i64,
+                    )),
+                    Some(KeyValue::new(
+                        opentelemetry_semantic_conventions::attribute::PROCESS_RUNTIME_NAME,
```

Comment:
could you also update the doc comment for this file to mention the new inclusion?

Reply:
done